OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -74,13 +79,19 @@ protected override void Created(Actor self)
 		{
 			orientation = self.Trait<BodyOrientation>();
 			targetableCells = self.TraitOrDefault<ITargetableCells>();
+			turret = self.TraitsImplementing<Turreted>().FirstOrDefault(t => t.Name == Info.Turret);
 
 			base.Created(self);
 		}
 
 		bool ITargetablePositions.AlwaysEnabled { get { return Info.RequiresCondition == null; } }
 
 		IEnumerable<WPos> ITargetablePositions.TargetablePositions(Actor self)
+		{
+			return TargetablePositions(self);
+		}
+
+		IEnumerable<WPos> TargetablePositions(Actor self)
```

Comment:
This can be merged back into `ITargetablePositions.TargetablePositions` above now.